### DISCUSSION TITLE
An Istallation Problem for WINDOWS OS

### DISCUSSION THREAD

**Discussion by LiamLech** [publicsamples/Quetzalcoatl#15] [Q&A] [answered]
Hello,

I put the vst3 file on C:\Program Files\Common Files\VST3
and put contents as the following image.
![スクリーンショット 2024-07-21 015649](https://github.com/user-attachments/assets/b0006ab1-8bab-4dfe-9680-74958d3a33ba)

Then the following messages still appear.
![スクリーンショット 2024-07-21 014604](https://github.com/user-attachments/assets/fe02b5d4-201c-4920-8bea-a6b6b87e9dbf)
![スクリーンショット 2024-07-21 014614](https://github.com/user-attachments/assets/3ec54b56-085f-497e-8f39-b4f5823b69ae)

**Comment by RickTaylor79** [comment, marked as answer]
Okay, you need to click on "Choose Sample Folder" and select C:\ProgramData\modularsamples\Quetzalcoatl\Samples

Then restart the plugin, it should work after this.

**Comment by LiamLech** [comment]
Thanks for your help. It works fine. q(≧▽≦q)

**Comment by potroub** [comment]
not working for me it always ask for a file with an extension but no where to be found...

**Comment by potroub** [comment]
I then just ignored messages about the .hr1 files... and it open it but no sound when I click to select a preset it freeze ablteon live

hopefully this is not made with JUCE because ableton has problem with it... so that means until it,s fixed it will be a no go for Ableton Live user 
Sorry...

**Comment by potroub** [comment]
i finally got it to work... but after 3 restart and 4 ableton live crashe...

it is very CPU consumming hopefully in the future that will be fixed thank you amazing synth

**Comment by RickTaylor79** [comment]
It can be a bit CPU intensive, what are your specs? The weakest CPU I've tested was a quad core 10th gen laptop i5 and it works okay there, playable with a buffer size of 512.

**Reply by potroub** [reply]
Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz
64.0 GB
buffer at 256 buffer size

**Reply by potroub** [reply]
but it works

**Reply by RickTaylor79** [reply]
> Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz 64.0 GB buffer at 256 buffer size

Thanks.. This is decent. Defiantly plan to improve performance, just not sure how much I can squeeze it. The stages menu reduces to number of samples in a patch, in you don't need all 8 this will improve things significantly,

**Comment by convinced-prog** [comment]
Went through all that rigmaroll and finally managed to get Bitwig to recognise 'Q' but struggled to display the gui.
Finally clicked something on the top menu (can't remember what) plus caps lock for some unknown reason, and the gui popped up in all it's glory, merrily flashing and blinking away to itself.
One question. How do I actually get it tro do anything?
I can load samples etc but at very low volume (better than nothing) Some of the controls don't do anything, all I can do is play the sample on my qwerty keyboard. At last something!
Is there something I'm missing as this thing looks really exciting and my ageing braincells are firing up.
Thanks for this

Just to let you know, I'm running on an Intel i5-6400 gaming pc, 16gb ram and 8gb vram with 2tb of storage. In Bitwig of course.

**Reply by convinced-prog** [reply]
Still can't figure out what part Caps Lock has in the plan of things because when I turn it off 'Q' goes silent?

**Reply by RickTaylor79** [reply]
It's probably a bitwig thing, the plugin doesn't respond to keyboard commands itself (would interfere with the way DAWs work). According to this Caps lock enables or disables midi notes from the computer keyboard:

https://www.bitwig.com/userguide/latest/user_interfacing/

As for the rest, I'm working on an update, should be released very shortly. Next hour or so.

**Reply by RickTaylor79** [reply]
This release should fix a few things for you, the unresponsive controls for sure:

https://github.com/publicsamples/Quetzalcoatl/releases/tag/0.6.5.1

**Reply by convinced-prog** [reply]
Great news. Now I have something to work with.
Will let you know how I get on.
Many thanks

**Comment by convinced-prog** [comment]
Hi
Just one problem here. The .dat file has disappeared and Cubase is asking
for it. Can't even find it on the web-site.
Any ideas?
Thanks

**Comment by RickTaylor79** [comment]
You can find it here:

https://github.com/publicsamples/Quetzalcoatl/releases/download/0.6.2/AudioResources.dat.zip

But it's not needed for 0.6.5, not sure why it's asking for it. You might need to clear your plugin cache..

https://acusticaudio.freshdesk.com/support/solutions/articles/35000145928-re-setting-the-cubase-plug-ins-cache-manually

**Comment by convinced-prog** [comment]
Done all that and plugin loaded instantly after rescan.
Thanks
Still not sure how to load the other sample files etc and still silent, hr1
files are a mystery to me and can't load them or not asked for.
Will fiddle a bit more.
Thanks for your help.

**Reply by RickTaylor79** [reply]
By the sounds of it you might need to trash your old content folder found at:

C:\ProgramData\modularsamples\Quetzalcoatl

Doing this will reset the plugin and you'll get the prompt to install the .hr1 file. Thanks for persevering!

**Comment by convinced-prog** [comment]
When you say Content folder do you mean all of it, including all the
samples etc?

**Reply by RickTaylor79** [reply]
Yep, the whole thing. It will repopulate once you've installed the content.

**Comment by convinced-prog** [comment]
Now it's complaining about the audio buffer again. I'm trying it out in
Bitwig. Have no clue how to set this

**Reply by RickTaylor79** [reply]
Screenshot of the error message? I don't know much about Bitwig, but I'm trying the demo and it works here. If you're having buffering issues try increasing the block size in your audio settings.

<img width="805" alt="Screenshot 2024-07-29 at 16 53 02" src="https://github.com/user-attachments/assets/403a24c7-440f-4aa7-aed5-a4f3c98214b4">

**Comment by convinced-prog** [comment]
THis what I get

**Reply by RickTaylor79** [reply]
Was there supposed to be an attachment? If you are responding via email it's better to use the GitHub page for attachments, I guess..

**Comment by potroub** [comment]
Ok I've falllowed your new instructions for the new way to insatll....

The install process seemed good...

But then When I copy each folder of Expansions in the FOLDER Calle EXPANSION

I loose sound I made previously and then... it freeeze the session in my daw....

Very Glitchy....

tell me when you think you can post the FOLDER how to put things where because it has probles

int the factory lib there is no presets

And then I lost old presets from expansion etc....

So... I can't even work on any thing because I loose all works and presets....
" (

Thank You
R

**Reply by potroub** [reply]
I wish you good luck because the synth looks realy cool and sounds nice

But I'm afraid I will never be able to use it in a daw session to be recalled properly and if it could use less CPU and stop freezin all the time we could get a nice patch and save it....

But for now... there is problems...
So I will wait for your synth to be curated from this anoying bugs thank you and wishing you luck

**Reply by RickTaylor79** [reply]
Sorry about this, but please keep in mind, it's a work in progress, beta etc. You have to expect stuff like this (from this amateur dev anyway!). For what it's worth, your old presets wouldn't have sounded the same because several things have changed since the last release. 

I've just posted an updated windows version that should/might improve performance somewhat:

https://github.com/publicsamples/Quetzalcoatl/releases/download/0.6.5.1/Quetzalcoatl.vst3.Win.zip

Right now performance varies considerably between DAWs, Reaper is really stable, as is Logic on the Mac side. Live is a fucking nightmare!  Pressing a note = 50% CPU consumption.. 


Thanks for checking it out all the same.

**Reply by potroub** [reply]
Let me try it out...

But when this beta sequence is done... it's an absolute must be able to recall previous session and previous sound design
Thanks
PeaceR I'll try the new VST to see...

**Reply by RickTaylor79** [reply]
Oh for sure, it definitely will. But the modulators work in a completely different way now (there are no longer dedicated mods for x-fade and filter) so it's in many ways not the same plugin v0.6.2 was. v0.7 will add even more new features, after 1.0 things will become stable. This particular plugin has been through so many iterations, just never seems to be quite _right_.

**Comment by convinced-prog** [comment]
Ok, thanks for the clarification.
Keep going Rick
All the best.
I still have faith in you and hope to get this performing,

**Reply by convinced-prog** [reply]
Hi
It does seem to be DAW specific. I don't give in easily so I tried it in Reaper and it dropped in without argument. Not being very savvy with Reaper I haven't figured out how to use it yet but you were right. I find Cubase too technical ( old braincells) and the other top DAWS are just the same.
Luckily I was able to find the initial loaded samples but have not as yet installed the others. How might I do this?
I don't care what DAW I use as long as I get the damn thing to work. I'm trying Oi Grandad today so wish me luck. My new midi keyboard arrives this week so more fun for the old man.
Please let me have your views on all this.
Many thanks again.

**Reply by RickTaylor79** [reply]
For me at least, Reaper is quite difficult to get to grips with, there are so many options.  it's very powerful though and the demo limitation is generous, so if you're looking to learn a DAW it's probably a good choice.

I'm gonna upload a video on how to install the plug-in and content later today, but this is the gist of it assuming the plugin is up and running:

1. Download and extract one or more of the expansion packs (there are 3 on release page, GM, Digital and Modular)
2. Navigate to C:\ProgramData\modularsamples\Quetzalcoatl\Expansions (ProgramData may [hidden by default](https://support.microsoft.com/en-gb/windows/view-hidden-files-and-folders-in-windows-97fbc472-c603-9d90-91d0-1166d1d9f4b5) on Win 11) and copy the expansion packs to this folder. So you'll end up with:

C:\ProgramData\modularsamples\Quetzalcoatl\Expansions\GM etc..

3. Start/Restart Quetzalcoatl 

Good luck with Oi, Grandad! It looks complicated, but really it's not, just poorly designed! Installation is the same as Q for the most part. Try to learn what's you can do with a single voice and modulator and then progress from there. They're for an older version, but there are some tutorial videos [here](https://www.youtube.com/watch?v=Sh98R5TAjUc&list=PL8Rp79UJ5uBC-SntUW93i3953yPq0DeSF&index=2) that might help you get started. 

Thanks for the support.

**Comment by convinced-prog** [comment]
Awesome, thanks for the info!
Reaper hasn't let me down! Oi Grandad is functioning, I think!!
Still don't understand how I can access the expansion packs, they remain as
archive type files in the expansion directory.
Now I'll have to figure out how to use Reaper.
Would you have any idea why 'Q' keeps telling me to re-load my samples
directory? It still doesn't quite run smoothly and without sound.
Anyway. Carry on!
Thanks, getting there

**Comment by RickTaylor79** [comment]
If they're archives, you'll need to unzip em!  if you're still having trouble, upload a screenshot directly to this [Github discussion](https://github.com/publicsamples/Quetzalcoatl/discussions/15#top) (click on the paper clip icon above the text box).

Here, I made some tutorial videos. the first deals with installation (Windows starts at around 2.08):

https://www.youtube.com/playlist?list=PL8Rp79UJ5uBCV9Uh7daNJhiqda7L4ccCv

**Reply by convinced-prog** [reply]
Thanks for all this but I still don't fully understand whats going on here. I don't understand how those library files are extracted and why Q is still wanting me to define my samples directory. I also noticed that you extracted the GM library file to a Music folder called Q. This stage isn't shown on the installation instructions.
I've slowed the video to 50% to help my braincells to catch up, perhaps I should slow it more?
Here we go again.

**Reply by convinced-prog** [reply]
Oi Grandad running well but Q is silent even thogh it is registering sound on the mixer.
On my head hurts!!!!!

**Comment by RickTaylor79** [comment]
Send screenshots of your C:\ProgramData\modularsamples\Quetzalcoatl & C:\ProgramData\modularsamples\Quetzalcoatl\Expansions folders and one of your Reaper project with Quetzalcoatl open.

To do this, don't reply by email. Visit the GitHub discussion page and click on the paper clip to upload your images. 

Better yet, email them to me. This thread is getting a bit nuts.

 modularsamples@gmail.com

**Comment by convinced-prog** [comment]
Here you go.
Still asking for samples folder and ignoring expansion samples
BTW is that Brazil nuts, monkey nuts or coconuts?

**Comment by convinced-prog** [comment]
![Screenshot (17)](https://github.com/user-attachments/assets/66df4ab2-ea14-4d0e-b611-02ff3a13b29e)
![Screenshot (18)](https://github.com/user-attachments/assets/3a2cc13a-78b1-499d-a364-c7e10f9c6f91)
![Screenshot (19)](https://github.com/user-attachments/assets/5bf69e1c-3f24-4b25-b1de-edf13d8fcad3)
![Screenshot (20)](https://github.com/user-attachments/assets/2209b5a7-9729-4642-a961-7d776fd43239)
![Screenshot (21)](https://github.com/user-attachments/assets/dfb7fc93-90c9-4a06-b7c3-aa5575e3df14)
![Screenshot (22)](https://github.com/user-attachments/assets/0eaaecd3-bf62-4849-a594-ef13ee10b0af)
As requested

**Comment by RickTaylor79** [comment]
Great. Okay, this is easy to fix!

The message "The audio buffer size should be a multiple of 8. Please adjust your audio settings" can be read literally. It means the plugin want's a buffer size of 8, 16, 32, 64. 128. 256, 512, 1024 etc.  Since you're on older hardware I'd suggest 512 or 1024. See [here for instructions](https://archive.steinberg.help/cubase_ai/v10/en/cubase_nuendo/topics/vst_system_link/vst_system_link_buffer_size_adjusting_t.html) on how to set you buffer size. 

Also, you've got your expansion folder setup incorrectly, place the extracted folder in expansions, so you'd have expansions/GM etc..

**Comment by convinced-prog** [comment]
Me again.
Placed GM files into /Expansions/GM.
Checked buffer size etc.
Opened Reaper in a new project. Inserted new instrument track 'Q'
Q immediately requested the samples folder, /Samples folder was empty
despite being fully populated on explorer, the same with /Expansions/GM.
Nothing but visible in files explorer.
Every time I try to access the samples I am requested to load them.
What do you think is going wrong?
Not my day!

**Reply by RickTaylor79** [reply]
That's definitely not right. I'd delete everything and start from scratch. This way you can be sure you have the most recent copy of everything.

**Comment by potroub** [comment]
Tried the new version and fallowed instructions with the video 

But still when I use Expansion and I try the preset from Digital it freezes all the time and I have to crash ableton

**Reply by RickTaylor79** [reply]
I just tried the presets in live and couldn't reproduce this.

Tell me, do you only see a single preset in the digital folder? If so you have an older version of the expansion. Grab the latest version. 

This plugin needs a package installer, too many variables right now..

**Comment by potroub** [comment]
I'll try to reproduce the bug with a video screen cap if I can do it...
busy day for now...


*Richard Le Gendre*
Ingénieur de son, SoundDesigner, Compositeur,
Réalisateur et Spécialiste Ableton Live.



*QuickMusic FactorySt**udio **Orange         *
135, Van Horne Montréal, Qc, Canada.
Tel : 514 282 0624
quickmusicfactory.ca
studioorange.ca
***@***.***

**Reply by RickTaylor79** [reply]
Cool. Thanks.

**Comment by RickTaylor79** [comment]
Since the original question here was about 0.6.2, and 0.6.5 has a different install method I'm gonna close this thread in the hope of avoiding confusion, and further triggering my latent OCD 😂. 

A new discussion for v0.6.5 Windows install problems can be found [here](https://github.com/publicsamples/Quetzalcoatl/wiki/Installation-v0.6). 

Please report crashes & bugs to the [issue tracker](https://github.com/publicsamples/Quetzalcoatl/issues), supplying system info and crash reports when relevant.
